pkg/controller/job/utils.go

```diff
@@ -29,3 +36,104 @@ func IsJobFinished(j *batch.Job) bool {
 	}
 	return false
 }
+
+func getCompletionsIndex(pod *v1.Pod) (int, error) {
+	return strconv.Atoi(pod.Labels[CompletionsIndexName])
+}
+
+func addCompletionsIndexToPodTemplate(job *batch.Job, completionsIndex int32) v1.PodTemplateSpec {
+	template := job.Spec.Template
+	if completionsIndex > 0 {
```

Comment:
return early here to avoid indentation of the whole `completionsIndex > 0` block.
```
if completionsIndex <= 0 {
	return template
}
```

Reply:
ok